docs/README.md

```diff
@@ -1330,43 +1330,135 @@ Read Models are classes decorated with the `@ReadModel` decorator that have one
+    readonly fieldA: SomeType,
+    readonly fieldB: SomeType,
+    /* as many fields as needed */
   ) {}
 
-  @Projects(Cart, 'id')
-  public static updateWithCart(cart: Cart, currentCartReadModel?: CartReadModel): CartReadModel {
-    return new CartReadModel(cart.id, cart.cartItems, cart.paid)
+  @Projects(SomeEntity, 'entityField')
+  public static projectionName(entity: SomeEntity, currentEntityReadModel?: NameReadModel): NameReadModel {
+    return new NameReadModel(/* initialize here your constructor properties */)
+  }
+  
+  @Projects(SomeEntity, 'othetEntityField')
+  public static projectionName(entity: SomeEntity, currentEntityReadModel?: NameReadModel): NameReadModel {
+    return new NameReadModel(/* initialize here your constructor properties */)
   }
+  /* as many projections as needed */
 }
 ```
 
 #### Read models naming convention
 
+As it has been previously commented, but semantics plays an important role in designing a coherent system and your application should reflect your domain concepts. Altough you have freedom to choose your read model names, we strongly recommend you to name them using the name of the entity that they are projecting followed by `ReadModel` suffix. 
+
+Despite you can place commands, and other Booster files, in any directory, we strongly recommend you to put them in `<project-root>/src/read-models`. Having all the commands in one place will help you to understand your application's capabilities at a glance.
+
+```text
+<project-root>
+├── src
+│   ├── commands 
+│   ├── common
+│   ├── config
+│   ├── entities
+│   ├── read-models  <------ put them here
+│   ├── events
+│   ├── index.ts
+│   └── read-models
+```
+
 #### Creating a read model
+
+The preferred way to create a read model is by using the generator, e.g.
+
 ```shell
 boost new:read-model CartReadModel --fields id:UUID cartItems:"Array<CartItem>" paid:boolean --projects Cart
 ```
 
-This will create a file in the read-models directory `<project-root>/src/read-models/CartReadModel.ts`.
+The generator will create a Typescript class under the read-models directory `<project-root>/src/read-models/CartReadModel.ts`.
 
 Read Model classes can also be created by hand and there are no restrictions regarding the place you put the files. The structure of the data is totally open and can be as complex as you can manage in your projection functions.
 
 #### The projection function
 A `Projection` is a method decorated with the `@Projects` decorator that, given a new entity value and (optionally) the current read model state, generate a new read model value.
 
-Read models can be projected from multiple [entities](#4-entities-and-reducers) as soon as they share some common key called `joinKey`.
+Read models can be projected from multiple [entities](#4-entities-and-reducers) as soon as they share some common key called `joinKey`. The `joinKey` is the field used across entities to relaunch the projection function. To clarify this let's see an example:
```

Comment:
To be honest I don't have clear the concept of the `joinKey` to explain it properly

Reply:
@rdiaz82 Does this answer from Javier help you to understand it?

(look at the part where he answers: _"I wonder whose our most prolific user– I should totally reach out to them"_)

https://spectrum.chat/boostercloud/general/data-access~4c5eccde-3e40-4282-92d4-9d3342dc0bbf?m=MTU5NjA1Nzc4NzU5OQ==

If not, we can have a meeting so that you can understand it and finis this part of the explanation 😉